### PR TITLE
change e2e-gcp-qe-no-capabilities optional

### PR DESCRIPTION
**Pull request by lance5890**
This PR changes etcd e2e-gcp-qe-no-capabilities ci job to optional

cc @tjungblu @Elbehery @fxierh

### CODE REVIEW

File: ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml [hunk 1]
```diff
@@ -35,6 +35,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-9-golang-1.20-openshift-4.16
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
```
Comment (tjungblu) [inline ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml:41]: master should probably be at 4.16 :)

**Comment by tjungblu** [comment]
/ok-to-test

@lance5890 you're also missing "make update", not sure you'll be able to access the builder images without RH credentials though.

File: ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml (hunk 1 above)
Comment (lance5890) [reply ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml]: done

**Comment by lance5890** [comment]
> /ok-to-test
> 
> @lance5890 you're also missing "make update", not sure you'll be able to access the builder images without RH credentials though.

I'm afraid not, maybe you can update on my branch, or close this PR, feel free.

**Comment by tjungblu** [comment]
@Elbehery can you incorporate the chances from Lance in your PR and re-generate please?

**Pull request closed without merging**